doc/api/process.md

```diff
@@ -1356,6 +1356,50 @@ In custom builds from non-release versions of the source tree, only the
 `name` property may be present. The additional properties should not be
 relied upon to exist.
 
+## process.resourceUsage()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object} containing resource usage measures for the current process
+
+Note: Windows only provides the following resource usage measures:
+
+- `userCpuTimeUsedSec`
+- `userCpuTimeUsedMs`
+- `systemCpuTimeUsedSec`
+- `systemCpuTimeUsedMs`
+- `blockInputOperations`
+- `blockOutputOperations`
+
+On Windows, all other values are `0`.
+
+On Linux, some fields may or may not be used/set to non-zero values depending on
+the kernel version. See the getrusage(2) man page for more information.
+
+Example return value:
+
+```js
+    { userCpuTimeUsedSec: 0,
+      userCpuTimeUsedMs: 159729,
+      systemCpuTimeUsedSec: 0,
+      systemCpuTimeUsedMs: 30597,
```

Comment:
`fooCpuTimeUsedUsec`?

Another option is a single `fooCpuTimeUsed` number with sub-second precision.  I think V8 should be able to unbox the double so it would be infinitesimally more efficient too (one slot instead of two.)